test/metric_value_benchmark/jmx_tomcat_jvm_test.go

```diff
@@ -46,7 +46,7 @@ func (t *JMXTomcatJVMTestRunner) GetAgentConfigFileName() string {
 }
 
 func (t *JMXTomcatJVMTestRunner) GetAgentRunDuration() time.Duration {
-	return 2 * time.Minute
+	return 10 * time.Minute
```

Comment:
Is 10 minutes necessary here or can we reduce it to 5 minutes or less? or is that still too short to collect this metric. Asking since sleeping 10 minutes for a test to run slow down the testing process.

Reply:
I think 5 is a good balance—will make this change and keep an eye on the tests